README.rst

```diff
@@ -118,16 +118,32 @@ The following multi node networking models are supported:
   - In conjunction with cloud network routers
```

Comment:
Could you replace the link for image `cilium_overview.png` to be a relative link instead of an absolute URL? That would allow us to preview future image changes.
```diff
-.. image:: https://cdn.rawgit.com/cilium/cilium/master/Documentation/images/cilium_overview.png
+.. image:: Documentation/images/cilium_overview.png
```
In the meantime, I've pushed a temporary branch to be able to review the image change (to check the new size specifically) at https://github.com/cilium/cilium/blob/pr/pchaigno/test/README.rst. It looks good.

Reply:
My impression was that this would ensure that in each stable branch we use the same image and wouldn't have to backport it?

Reply:
I've added it to open session for later today.

Reply:
Do we really care about the README in stable branches? And if we do, do we really want e.g. the v1.8 README to say Cilium supports bandwidth management?

Reply:
No, I don't think we care about the README in stable branches.